db/db_block_cache_test.cc

```diff
@@ -1263,6 +1275,134 @@ INSTANTIATE_TEST_CASE_P(
         ::testing::Values(PinningTier::kNone, PinningTier::kFlushedAndSimilar,
                           PinningTier::kAll)));
 
+#if (defined OS_LINUX || defined OS_WIN)
+// Since FS_IOC_GETVERSION is not supported with tmpfs, GetUniqueId returns
```

Comment:
Is this true even with the recent changes to use DB session ID if GetUniqueId is not supported?

Reply:
With DB session ID it works and return same key prefix. I will update the test.